pkg/serverconfig/types.go

```diff
@@ -96,6 +98,12 @@ type DeveloperConsoleCatalogCategory struct {
 	Subcategories []DeveloperConsoleCatalogCategoryMeta `json:"subcategories,omitempty" yaml:"subcategories,omitempty"`
 }
 
+// AddPage allows customizing the actions on the Add page in developer perspective.
+type AddPage struct {
+	// disabledActions is a list of action IDs that are not shown to users.
```

Comment:
Comments should match those in console-operator https://github.com/openshift/console-operator/blob/e2379446a9accf52881166499f34be0d2e104296/pkg/console/subresource/consoleserver/types.go#L111-L116

Reply:
Updated all docs. Thanks. :+1: